src/index.js

```diff
@@ -59,7 +60,8 @@ export default class KeyboardAccessory extends Component {
 
   render() {
     let { bottom } = this.state;
-    let { children, backgroundColor } = this.props;
+    let { children, backgroundColor, verticalOffset } = this.props;
+    bottom = bottom + verticalOffset;
```

Comment:
I just checked the snap and checked if I could make it work with this patch. I needed to make the following change to make it work completely. Otherwise the input would be moved down even when no keyboard was present, effectively moving the input underneath the tabbar.
```suggestion
    if (bottom != 0) {
      bottom = bottom + verticalOffset;
    }
```